### ISSUE TITLE
When using Container as Body, placement of the menu is too low

### ISSUE THREAD

**Issue by ethanhunt123** [silviomoreto/bootstrap-select#739]
Hello,

When using the setting container = 'body' the placement of the menu is too low. It caused by the code of line 474 (or there abouts):

actualHeight = $element.hasClass('dropup') ? 0 : $element[0].offsetHeight;

This ends up adding extra padding to above the menu. My guess is this is here because if you don't do this, when clicking outside the menu on the html without selecting a value, the following is called on 492:

 $('html').on('click', function (e) {
           $('html').on('click', function (e) {
        if ($(e.target).closest(that.$newElement).length < 1) {
          $drop.removeClass('open');
        }
      });

This is to close any open menus. The issue is that if you dont add to the offset of the height, this cause you not to be able to select the menu again when removing the open class (i think its  a focus issue).

This can be solved by changing that event to:

 $('html').on('click', function (e) {
          if ($(e.target).closest(that.$newElement).length < 1) {
              //CHANGE
              if ($drop.hasClass('open')) {
                  that.$menu.detach();
                  $drop.removeClass('open');
                  $drop.detach();
          }
        }
      });

So to summarize, my suggestion is to change:

Line 474 to: 

actualHeight = 0;

and Line 492 to:

 $('html').on('click', function (e) {
          if ($(e.target).closest(that.$newElement).length < 1) {
              //CHANGE
              if ($drop.hasClass('open')) {
                  that.$menu.detach();
                  $drop.removeClass('open');
                  $drop.detach();
          }
        }
      });
